spec/features/user_downvotes_on_review_spec.rb

```diff
@@ -7,37 +7,34 @@
 } do
 
   describe 'User downvotes on a review' do
-    scenario 'user clicks downvote' do
+    scenario 'user clicks downvote', js: true do
       review = FactoryGirl.create(:review)
       beach = review.beach
       user = review.user
 
       sign_in(user)
 
-      visit '/'
-      click_on beach.name
+      visit beach_path(beach)
+
       click_on 'Downvote'
+      sleep 1
 
-      expect(page).to have_content("Downvote created successfully.")
-      expect(page).to_not have_content("Downvote not created successfully.")
-      review.reload
       expect(review.score).to eq(-1)
     end
-    scenario 'user clicks remove downvote' do
+    scenario 'user clicks remove downvote', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.